Automattic/wp-calypso
# Launchpad: Migrate plan_selected  to use the new structure on free flow

Continue the work related to #86432
Depends on #86491

!Importante!:  This PR is the last one covering the free flow, so run the free flow is importante here

## Proposed Changes
Steps migrated: 
*  plan_selected


The new step definition will only by available on the free flow for testing purposes. 

## Testing Instructions
* Start the setup/free. flow until reach the ` setup/free/launchpad?`  page
* Enable the toggle adding the param  `&flags=launchpad/new-task-definition-parser`  on the page url 
* Check the Webtools console to see info logs describing if the tasks above are running new version. 
Something similar ` Using new task definition parser `  + task name
* Check if you can complete the full flow with the feature toggle ON. Please pay attention that some steps are remoging the params from the url when they are completed. 



## Pre-merge Checklist
